tools/myst/enc/syscall.c

```diff
@@ -900,9 +900,9 @@ static long _get_mempolicy(
     unsigned long flags)
 {
     long ret;
-    size_t maxnode_byte = maxnode / 8;
+    size_t nodemask_size = maxnode / 8;
```

Comment:
Using sizeof(...) instead of hardcoding of 8 would be better

Reply:
using CHAR_BIT here since 8 is the translation of bits to byte, and is used as `8` in the spec as well